### PR TITLE
relocate imports

### PR DESCRIPTION
**Pull request by tdincer**
This PR moves the module imports from the functions to the top of the file. This is more efficient as the modules will be imported only once. In the existing configuration, modules are imported for each key in each table.

### CODE REVIEW

**Comment by CBroz1** [comment]
Is this impacting runtime in production? I thought [modules weren't loaded twice](https://stackoverflow.com/questions/19077381/what-happens-when-a-module-is-imported-twice).

Even so, I feel like we could find a more clever middle ground. Some DLC functions take a solid 10-20s to load on my machine, which I would want to avoid in an instance where a user isn't going to interact with that `make` method. Is there a way to check the namespace to see if they're loaded and only import if not?

**Comment by tdincer** [comment]
It's impacting the runtime in production. Modules are imported as many times as they the functions are called, if the modules are imported only inside the functions.

Waiting a solid 10-20s once is fine but the existing configuration waits 10-20s every time a make function is called (once per key).

Try this:
```python
import sys

def np_sqrt(x):
    import numpy as np
    return np.sqrt(x)

'numpy' in sys.modules.keys()
```

This will return False because numpy is not accessible outside the function.

**Comment by CBroz1** [comment]
Ah, ok. I understand now. Thanks for explaining. Please update the changelog to reflect new strategy and version bump

**Comment by dimitri-yatsenko** [comment]
Placing the imports inside the function can be better when the function call is rare and the imports or only needed in one place. This speeds up the initial import.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+## [0.2.1]
++ Relocate module imports to the top of the files
+
 ## [0.2.0] - Unreleased
```
Comment (CBroz1) [inline CHANGELOG.md:9]: I'm sorry, I didn't realize the previous version was unreleased.

```suggestion
## [0.2.0] - Unreleased
+ Relocate module imports to the top of the files
```

File: element_deeplabcut/version.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
 """
 Package metadata
 """
-__version__ = "0.2.0"
+__version__ = "0.2.1"
```
Comment (CBroz1) [inline element_deeplabcut/version.py:4]:
```suggestion
__version__ = "0.2.0"
```

**Comment by tdincer** [comment]
I'm dealing with several 100s of calls

**Review by CBroz1 — APPROVED** [review]

**Review by CBroz1 — APPROVED** [review]

**Pull request merged**